oidc-exchange.py

```diff
@@ -0,0 +1,65 @@
+import os
+import sys
+
+import requests
+
+
+def die(msg: str):
```

Comment:
I have this idea recorded https://github.com/pypa/gh-action-pypi-publish/issues/106. It's not explained there but I wanted to post some results to the job summary. It'll be more colorful and you can use Markdown. It's implemented by appending to a certain file.
I think you could start doing so in this PR, by adding the same output that you print on the console.
You can check out my `alls-green` action to get an idea of how it's done.

Reply:
Sounds good! Will do.